sphinx/src/maintainer/knowledge_base.rst

```diff
@@ -1939,7 +1939,7 @@ You are also free to download recipes and rebuild them yourself, if you would li
    A bot then assesses that the submitting feedstock has permission to build the package it has submitted, and only then will it relay the build to the ``conda-forge`` channel.
    This helps mitigate against a bad actor gaining access to an inconspicuous feedstock and then trying to push a build with malicious code into essential infrastructure packages (e.g., OpenSSL or Python).
 3. We have `artifact-validation <https://github.com/conda-forge/artifact-validation>`__ for validating all the conda-forge artifacts uploaded to ``anaconda.org``. This validation scans for various security-related items, such as artifacts that overwrite key pieces of certain packages.
-4. We have a dedicated :ref:`Security and Systems Sub-Team <security_subteam>` who works hard towards making sure to secure and maintain appropriate access to the credentials and services/systems used by conda-forge.
+4. We have a dedicated `Security and Systems Sub-Team <security_subteam>` who works hard towards making sure to secure and maintain appropriate access to the credentials and services/systems used by conda-forge.
```

Comment:
This kind of change won't work. Needs to be a href as described above.

Reply:
Fixed